requirements.txt

```diff
@@ -4,6 +4,6 @@ duckcli==0.0.1
 # Database adapter
 dbt-duckdb>=1.1.2,<1.2.0
 
-# Look for `profiles.yml` in the current working directory to avoid needing to `export DBT_PROFILES_DIR=.`
-git+https://github.com/dbt-labs/dbt-core.git@experiment/local-profiles-1.1.1#egg=dbt-core&subdirectory=core
-git+https://github.com/dbt-labs/dbt-core.git@experiment/local-profiles-1.1.1#egg=dbt-postgres&subdirectory=plugins/postgres
+# extra features
+
+sqlfluff~=1.2.1
```

Comment:
gotta bring this back for the environment tests

```suggestion
 Look for `profiles.yml` in the current working directory to avoid needing to `export DBT_PROFILES_DIR=.`
git+https://github.com/dbt-labs/dbt-core.git@experiment/local-profiles-1.1.1#subdirectory=core
git+https://github.com/dbt-labs/dbt-core.git@experiment/local-profiles-1.1.1#subdirectory=plugins/postgres

# extra features

sqlfluff~=1.2.1
```